src/org/parosproxy/paros/view/MainFrame.java

```diff
@@ -658,7 +658,8 @@ public void setShowTabNames(boolean showTabNames) {
 	 */
 	public void setTitle(Session session) {
 		StringBuilder strBuilder = new StringBuilder();
-		strBuilder.append(session.getSessionName()).append(" - ");
+		String sessionName = session == null ? "" : session.getSessionName();
+		strBuilder.append(sessionName).append(" - ");
 		if (!session.isNewState()) {

```

Comment:
I should trust myself more. I did it this way, then I tweaked it to account for null on this too. Then I read the doc on isNewState and thought 'otherwise false' likely already handles null session and undid my revised change...

Will update this PR when I get into the office....On my way now.